fe/src/components/Icons/BlocksLeft.vue

```diff
@@ -0,0 +1,196 @@
+      fill="#894727"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M98.7612 66.8308L65.7999 54.4265L62.5388 23.0958L95.5001 35.5001L98.7612 66.8308Z"
+      fill="#9D512D"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M98.761 66.8308L128.461 47.9044L125.2 16.5736L95.4999 35.5001L98.761 66.8308Z"
+      fill="#C46538"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M125.2 16.5736L92.2389 4.16932L62.5388 23.0958L95.5 35.5001L125.2 16.5736Z"
+      fill="#B05B32"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M63.7605 92.7704L51.6265 79.6264L33.8308 81.4473L45.9649 94.5913L63.7605 92.7704Z"
+      fill="#3B1E11"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M63.7606 92.7704L51.6265 79.6264L57.2881 64.6616L69.4222 77.8055L63.7606 92.7704Z"
+      fill="#763D22"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M51.6265 79.6264L33.8309 81.4473L39.4925 66.4824L57.2881 64.6616L51.6265 79.6264Z"
+      fill="#763D22"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M45.9649 94.5913L33.8309 81.4473L39.4924 66.4824L51.6265 79.6264L45.9649 94.5913Z"
+      fill="#9D512D"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M45.9649 94.5913L63.7606 92.7704L69.4221 77.8055L51.6265 79.6264L45.9649 94.5913Z"
+      fill="#9D512D"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M69.4222 77.8055L57.2881 64.6616L39.4925 66.4824L51.6265 79.6264L69.4222 77.8055Z"
+      fill="#C46538"
+      stroke="#954824"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M66.1439 217.647L34.6049 207.605L7.64746 226.811L39.1864 236.853L66.1439 217.647Z"
+      fill="#4E2816"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M66.144 217.647L34.605 207.605L30.0234 178.357L61.5624 188.399L66.144 217.647Z"
+      fill="#763D22"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M34.6049 207.605L7.64746 226.811L3.06592 197.562L30.0233 178.357L34.6049 207.605Z"
+      fill="#894727"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M39.1864 236.853L7.64746 226.811L3.06592 197.562L34.6049 207.605L39.1864 236.853Z"
+      fill="#9D512D"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M39.1865 236.853L66.144 217.647L61.5624 188.399L34.605 207.605L39.1865 236.853Z"
+      fill="#C46538"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M61.5623 188.399L30.0233 178.357L3.06592 197.562L34.6049 207.605L61.5623 188.399Z"
+      fill="#B05B32"
+      stroke="#9C441B"
+      stroke-miterlimit="1"
+    />
+  </svg>
+</template>
+
+<script>
+export default {
+  name: "BlocksLeft"
+};
```

Comment:
```suggestion
```
Nuxt has auto import feature based on filename under components directory.
This is no longer needed.
Please remove.